Extending/Key-Vault/index-v10.md

```diff
@@ -0,0 +1,74 @@
+---
+versionFrom: 10.0.0
+meta.Title: "Azure Key Vault"
+meta.Description: "A guide configuring an Azure Key Vault"
+---
+
+# Configuring an Azure Key Vault
+
+From a security perspective, it's always a good solution to store your application secrets (such as a connection string and others keys) in an Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault
+
+## Installing the package
+
+Before you begin, you need to install the `Azure.Extensions.AspNetCore.Configuration.Secrets` and the `Azure.Identity` NuGet packages. There are two approaches to installing the packages:
+
+1. Use your favorite IDE and open up the NuGet Package Manager to search and install the packages
+1. Use the command line to install the package
+
+## Installing through command line
```

Comment:
Looks to me like a subheading to the "Installing the package" section? I could be wrong though. What do you think?

```suggestion
### Installing through command line
```